### ISSUE TITLE
Some questions and suggestions about VFXEDITOR Api4 version

### ISSUE THREAD

**Issue by yangjimzero** [0ceal0t/Dalamud-VFXEditor#11]
1. I think it will be more convenient to move the enabled switch of emitter to this position, because it is often necessary to switch repeatedly to preview a single emitter (even the binder option)
![image](https://user-images.githubusercontent.com/71748363/134680307-4581ed3d-2f63-486d-845c-9b2c8f8aad16.png)
2. In addition, I don't know why the enabled switch of particle doesn't work correctly (so I usually change "create count" to 0 as a substitute). I hope it can be repaired. In addition, maybe it is also placed in front of the list by its position?
![image](https://user-images.githubusercontent.com/71748363/134681066-e462d127-74a8-4411-aaa3-20ad4ab7579d.png)
3. Could you please do a bind point ID guide? Although I have studied the corresponding relationship between most IDs and skeletons, it will be more convenient for most other producers to have a guide.
![image](https://user-images.githubusercontent.com/71748363/134681763-00ea8ee0-1555-48f1-8fa0-f525f954db8a.png)

In addition，Is there any way to import the ". MDL" model extracted from the game? Or is there a way to convert MDL to gltf? I studied it, but I couldn't find a feasible way.

(Have a distant dream： I now use WinHex to modify TMB (modify some. PAP and. SCD referenced by it), and textools to modify skill icons. Finally, I repackage it with the modified VFX. It's more troublesome. I hope that one day I can integrate the operation directly in the editor.)

Finally, thank you for your continuous updates and contributions and for taking my suggestions.

**Comment by XCZA** [comment]
Yeah as a creative who love making tons of cool effects, its absolutely painful how how it is to single out certain FX. In addition to what they mentioned about the Emitter 'enable' function not working, setting the duration to 0 sometimes doesnt truely work, since the certain fx with flashes are rendered for just one instant. 

If there was a button like this, it would be incredibly helpful
![image](https://user-images.githubusercontent.com/112023493/201994327-f0488a92-6bc7-42c2-8b85-2dad41b804e8.png)
